### ISSUE TITLE
malformed url

### ISSUE THREAD

**Issue by sebyx07** [emberjs/data#4471]
when i try to pass arguments to queries to the store or find, the url is malformed

```js
this.store.findRecord('organization', params.name, {include: ['users']});
```

```
http://localhost:4200/api/v1/organizations/1234?{%22include%22:[%22users%22]}
```

Ember Data 2.7.0-beta.2

**Comment by fivetanley** [comment]
Was this different than earlier versions of Ember Data? It looks like URL parameters are just getting URL encoded.

**Comment by pangratz** [comment]
Thanks for reporting, this has been fixed in #4466...
